languages/elixir/concepts/access-behaviour/about.md

```diff
@@ -1 +1,57 @@
-TODO: add information on access-behaviour concept
+Elixir uses [_Behaviours_][behaviours] to provide common generic interfaces while facilitating specific implementations for each module which implements the behavior. One of those behaviours is the _Access Behaviour_.
```

Comment:
I have rewritten the first part of this to focus more on the access behaviour, and less on behaviours in general. I'm hoping we will manage to get behaviours as its own concept done.